keyman_trust_darwin.go

```diff
@@ -61,11 +62,12 @@ func (cert *Certificate) AddAsTrustedRootIfNeeded(elevatePrompt, installPromptTi
 	// Add it as a trusted cert
 	cmd = elevatedIfNecessary(elevatePrompt)("security", "add-trusted-cert", "-d", "-k", OSX_SYSTEM_KEYCHAIN_PATH, tempFileName)
 	out, err := cmd.CombinedOutput()
-	if installAttempted != nil {
-		installAttempted(err)
-	}
 	if err != nil {
-		return fmt.Errorf("Unable to run security command: %s\n%s", err, out)
```

Comment:
@oxtoacart - this means that installAttempted will not be called on successful install

Reply:
Doh! Of course, let me fix.